jinahub/indexers/query/vector/AnnoyIndexer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jinaai/jina:1.1.10
+FROM jinaai/jina:2.0.0
```

Comment:
I don't think this one exists

Reply:
This is the same with numpy indexer. 
https://github.com/jina-ai/indexers/blob/69279e78f2e437e50a23270e915abc3d7c4a7499/jinahub/indexers/query/vector/NumpyIndexer/Dockerfile#L1

Reply:
Yeah I also doubt this, I will try later

Reply:
sure, already fixed I think